### ISSUE TITLE
Training Custom Prediction Issues

### ISSUE THREAD

**Issue by lukeseed** [OlafenwaMoses/ImageAI#500]
Hi.

I'm trying to train a custom prediction class with, what looks to me to be, clearly differentiated classes.
Signal Jpg
![signal](https://user-images.githubusercontent.com/12753558/76110577-956cfb00-5f93-11ea-93de-d9bf85d92cdf.jpg)
Ambient Jpg
![ambient](https://user-images.githubusercontent.com/12753558/76110580-969e2800-5f93-11ea-9098-3c0218c11c15.jpg)

I've set my training up with the following
model_trainer.trainModel(num_objects=2, num_experiments=100, enhance_data=True, batch_size=64, show_network_summary=True, training_image_size=512)

I have the following ratios (train/test) of training images: signal(1,211/400) ambient(6,600/600)

After 1 experiment I ended up with the following losses:
122/123 [============================>.] - ETA: 6:42 - loss: 0.0868 - acc: 0.9809 
16/16 [==============================] - 494s 31s/step - loss: 6.4945 - acc: 0.6000
After 2 experiments the loss did not improve so I didn't continue past that.

When I go to do custom prediction on another set of images, I'm getting 100% ambient detection and e-30% signal detection.

Any ideas on where I went wrong?
Thanks for the support.

**Comment by skoroneos** [comment]
Maybe its an overfitting issue. 
Try to keep both classes at about the same number of samples.
Also is the training_image_size correct in both cases ?
